app/main/posts/modify/post-media.directive.js

```diff
@@ -91,7 +104,7 @@ function (
 
             function deleteMedia(mediaId) {
                 // Mark for deletion
-                $scope.media = {};
+                $scope.media = {id: mediaId};
```

Comment:
not a blocker if you can't do it now, but could you move changed and deleted to inside this declaration? since they are the next two lines?

Reply:
Heh, yes, i'll fix it now